tomcat/src/main/java/org/apache/coyote/support/FileUtils.java

```diff
@@ -0,0 +1,16 @@
+package org.apache.coyote.support;
+
+public class FileUtils {
+
```

Comment:
```suggestion
public final class FileUtils {
    private FileUtils() {}
```

유틸성 클래스는 인스턴스가 생성되는 경우를 원천차단하도록
final 키워드와 private 생성자를 선언해주면 좋습니다.